### ISSUE TITLE
Installation ignores user defined `PUB_CACHE`

### ISSUE THREAD

**Issue by royarg02** [pingbird/puro#25]
Noticing that puro creates an entry for `PUB_CACHE` in my `.profile`, I tried to reinstall it after setting `PUB_CACHE="~/.cache/pub-cache"`.

However, puro still creates `$PURO_ROOT/shared/pub-cache`, with all the related contents inside it.

I would like the tool to honor the environment variable.

```sh
[i] Puro 1.3.2 (distribution/linux-x64)
    Dart 2.19.5 (stable) (Unknown timestamp) on "linux_x64"
```

**Comment by pingbird** [comment]
Puro doesn't create `$PURO_ROOT/shared/pub-cache`, the pub tool does, if you change `~/.profile` try logging out to ensure it takes effect everywhere.

**Comment by royarg02** [comment]
I had made sure that the changed variable takes effect; the official flutter tool uses the defined location but puro does not.

**Comment by pingbird** [comment]
I found the issue, Puro was overriding PUB_CACHE in both .profile and when spawning a child process, this should be fixed in 1.3.3

Thanks!
